mergify_engine/web/api/queues.py

```diff
@@ -317,5 +324,435 @@ async def repository_queues(
                 )
 
             queues.queues.append(queue)
-
         return queues
+
+
+QueuePauseSchema = voluptuous.Schema({voluptuous.Required("message"): str})
+
+
+@router.post(
+    "/repos/{owner}/{repository}/queue/{queue_name}/pause_from",  # noqa: FS003
+    summary="Pauses merge queue(s)",
+    description="Pauses the merge of all merge queues equal to and under the requested queue priority level",
```

Comment:
```suggestion
    description="Pauses the merge of a queue and the queues following it",
```
There's no "priority" across queues per se, just an order of processing. Priority is confusing as it's used to order PR inside the queue.